components/searchInput.vue

```diff
@@ -157,7 +163,22 @@ export default {
   computed: {
     ...mapState({
       courseTag: state => state.courseTag
-    })
+    }),
+    tableData() {
+      const tabData = []
+      for (const data of this.studentsData) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Feito, mas tive que fazer uma deep copy, e, atualmente, eu só sei fazer pelo JSON.  Ainda uso isso como referência: https://dev.to/rsschouwenaar/how-to-make-a-real-copy-of-a-javascript-array-with-objects-without-a-reference-5md